### PR TITLE
Add com.toolstack.Folio

### PR DESCRIPTION
**Pull request by toolstack**
### Please confirm your submission meets all the criteria

A pretty note-taking app for GNOME (forked from Paper).

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: com.toolstack.Folio.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.toolstack.Folio",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=host"
```
Comment (hfiguiere) [inline com.toolstack.Folio.json:12]: this is a very broad filesystem permission. is it necessary?

File: com.toolstack.Folio.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            }]
+        },
+        {
+            "name" : "Folio",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [{
+                "type" : "git",
+                "url" : "https://github.com/toolstack/Folio",
+                "branch" : "main"
```
Comment (hfiguiere) [inline com.toolstack.Folio.json:43]: never use git branch. You must use a tag or/and commit

File: com.toolstack.Folio.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name" : "blueprint-compiler",
+            "buildsystem" : "meson",
+            "sources" : [{
+                "type" : "git",
+                "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler.git",
+                "branch" : "v0.10.0"
```
Comment (hfiguiere) [inline com.toolstack.Folio.json:33]: never use a branch for git.
Also it's missing the cleanup rule to remove it all.

File: com.toolstack.Folio.json (hunk 2 above)
Comment (toolstack) [reply com.toolstack.Folio.json]: Doh, sorry, grabbed the wrong version, fixed.

File: com.toolstack.Folio.json (hunk 1 above)
Comment (toolstack) [reply com.toolstack.Folio.json:12]: Yes, the user might want to change where their notes are stored.  Might be able to limit it to user directory though.

File: com.toolstack.Folio.json (hunk 3 above)
Comment (toolstack) [reply com.toolstack.Folio.json]: Done.

**Comment by hfiguiere** [comment]
bot, build com.toolstack.Folio

**Comment by toolstack** [comment]
@hfiguiere any suggestion on the build error?  I have screenshots included so I'm not sure what it's complaining about.

**Comment by hfiguiere** [comment]
have you check they actually are screenshot **images** ?

**Comment by toolstack** [comment]
That was my first thought so I did change them over to the raw github url just in case (my current attempts at resolving can be found [here](https://github.com/toolstack/Folio/blob/bd5d2d2ad1bf1925d3ba44451885bafa5c1f8704/data/app.metainfo.xml.in#L277)), but it didn't make a difference.

**Comment by bbhtt** [comment]
Use `raw.githubusercontent` links, drop the thumbnail images those are autogenerated. Then update the commit here.

**Comment by bbhtt** [comment]
This commit seems fine https://github.com/toolstack/Folio/commit/12dec0322c33aed2adc6babc082a192f33a630b7

**Comment by bbhtt** [comment]
bot, build com.toolstack.Folio

**Comment by toolstack** [comment]
Done, we'll see if it works, when I linted that locally it still gave the screenshots error 🤷

**Comment by toolstack** [comment]
@bbhtt thanks, that worked thought it still throws the error on my local linting, must be something on my local setup.

**Pull request closed without merging**